packages/bridge-ui-v2/src/components/Bridge/Amount.svelte

```diff
@@ -33,17 +37,19 @@
   let inputBox: InputBox;
   let computingMaxAmount = false;
 
+  onMount(() => {
+    clearAmount();
+  });
+
   // Public API
   export function clearAmount() {
     inputBox.clear();
     $enteredAmount = BigInt(0);
   }
 
   export async function validateAmount(token = $selectedToken, fee = $processingFee) {
-    $insufficientBalance = false;
-    $insufficientAllowance = false;
```

Comment:
Is this fixing a problem? We're gonna validate these two, we start assuming there is no issue and then validate

Reply:
yes, it helps with the flickering of the buttons. In the future we could introduce a loading or disabled status